src/index.html

```diff
@@ -1,14 +1,235 @@
+            <p class="menue__item__restaurant">To Be</p>
+            <ul class="menue__item__tag double__tag">
+              <li>
+                 Японська
+              </li>
+              <li>
+                 Азіатська
+              </li>
+              <li>
+                 Суші
+              </li>
+            </ul>
+            <p class="menue__item__time"> 25-35 Min </p>
+          </div>
+          <div class="menue__item">
+            <img src="images/menue__items__img/95b745f6ce6a45903c3ed254afd72e46-w550-279.png" alt="">
+            <p class="menue__item__restaurant">Druzi Cafe (вул. Прорізна)</p>
+            <ul class="menue__item__tag double__tag">
+              <li>
+                 Бургери
+              </li>
+            </ul>
+            <p class="menue__item__time"> 30-40 Min </p>
+          </div>
+
+        </div>
+      </div>
+    </div>
+  </main>
+  <footer>
+    <div class="box-content">
+      <div class="footer__logo">
+        <a href="#" class="logo">
+          uber <span>eats</span>
+        </a>
+      </div>
+      <div class="footer__info">
+        <ul>
+            <li class="select__lang">
+              <img class="globe" src="images/icon/Shape1.svg" alt="">
+              <select class="" name="">
+                <option value="Eng">ENGLISH</option>
+                <option value="Ua">УКРАЇНСЬКА</option>
+              </select>
+              <img class="arr__down" src="images/icon/Arr__down.png" alt="">
+            </li>
+            <li class="logo__tag">
+              <a href="#"><span>#</span>UberEats</a>
+            </li>
+            <li class="soc__tag">
+              <a href="#"><img src="images/icon/facebook__icon.svg" alt=""></a>
+              <a href="#"><img src="images/icon/tweeter__icon.svg" alt=""></a>
+              <a href="#"><img src="images/icon/instagram__icon.svg" alt=""></a>
+            </li>
+          </ul>
+          <ul>
+            <li>
+              <a href="#">About UberEats</a>
+            </li>
+            <li>
+              <a href="#">Become a Delivery Pa</a>
+            </li>
+            <li>
+              <a href="#">Become a Restaurant</a>
+            </li>
+        </ul>
+        <ul>
+          <li>
+            <a href="#">See all cities</a>
+          </li>
+          <li>
+            <a href="#">Pricing</a>
+          </li>
+          <li>
+            <a href="#">Help</a>
+          </li>
+          <li>
+            <a href="#">FAQs</a>
+          </li>
+        </ul>
+
+      </div>
+      <div class="footer__stores">
+        <a href="#"><img src="images/Stores__img/en_badge_web_generic-cf6dad406f.png" alt=""></a>
+        <a href="#"><img src="images/Stores__img/Group.png" alt=""></a>
+      </div>
+      <div class="footer__end">
+        <a href="#">© 2017 Uber Technolo</a>
+        <a href="#">Privacy</a>
+        <a href="#">Terms</a>
+      </div>
+      <a href="#back_to_top" class="button__up">
```

Comment:
place this on the bottom on the page and don't stick it, because it covers other content